nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -986,11 +981,8 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 				}
 				else
 					LOG_DEBUG(L"Error in calling fillVBuf");
-				childPacc->Release();
-				VariantClear(&(varChildren[i]));
+				child.Clear();

```

Comment:
All the calls to child.clear are not necessary I think, as CComVariant's destructor does this itself.